package.json

```diff
@@ -9,7 +9,7 @@
     "lint": "standard",
     "test": "npm run test:node && npm run test:browser",
     "test:browser": "karma start karma.conf.js",
-    "test:node": "tape ./tests/index.js",
+    "test:node": "node tests/",
```

Comment:
Wasn't tape supposed to work? I guess it wasn't. I think it should also specify the file to be explicit.